src/patternfly/components/TextInputGroup/templates/text-input-group--chip-group.hbs

```diff
@@ -1,143 +1,143 @@
-{{#> chip-group chip-group--id=(concat text-input-group--id '-chip-group')}}
```

Comment:
I'm assuming the logic having the `--id` set at the level of the chip group is in case this file includes multiple chip groups - each would be able to pass a unique `--id` to its chips. Using `text-input-group--id` on the chips would end up with dupes if there was more than one chip group.